.gitignore

```diff
@@ -21,4 +21,5 @@ docs/_build
 .*.swp
 **/.docker/
 **/.env
+**/.factorypath
```

Comment:
Where is this created?

Reply:
I somehow got it when update Maven Project using IDE Maven plugins